apis/metal3.io/v1alpha1/baremetalhost_validation.go

```diff
@@ -50,16 +58,38 @@ func (host *BareMetalHost) validateChanges(old *BareMetalHost) []error {
 	return errs
 }
 
-func validateRAID(r *RAIDConfig) error {
-	if r == nil {
-		return nil
+func validateBMCAccess(s BareMetalHostSpec, bmcAccess bmc.AccessDetails) []error {
```

Comment:
Maybe the name of this function is inappropriate, not sure what it should be called.